nengo_gui/components/value.py

```diff
@@ -59,6 +59,9 @@ def update_client(self, client):
         # while there is data that should be sent to the client
         while len(self.data) > 0:
             item = self.data.popleft()
+            while len(self.data) > 0:
```

Comment:
Hmm, I suppose that does look a little weird.  What I could do instead is get rid of the ```deque``` and just keep the concatenated list of binary data to send....